docs/api/browser-window.md

```diff
@@ -1356,6 +1356,14 @@ removed in future Electron releases.
 **Note:** The BrowserView API is currently experimental and may change or be
 removed in future Electron releases.
 
+#### `win.setForwardMouseMessages(forward)` _Windows_
```

Comment:
> Examples are always good. Where would the example belong?

You can make it a sub-chapter of `Click-through window` in `frameless-window.md`.